Documentation/adding-new-repository.md

```diff
@@ -41,6 +41,30 @@ When applicable, prepare Docker image repositories for the new component.
   * robots: Read & Write
 * For closed source repository, create a new private repository in registry.mender.io
 
+## Makefile
+
+Consider adding a Makefile to the repository. This will serve both developers on their
+local environment and automated templated jobs on CI.
+
+A Makefile in a Mender repository is expected to contain at least the following targets:
+
+* `fmt`: Format the code. Typically using `gofmt` for golang repositories
+* `lint`: Lint the code. On new golang repositories, we are trying to use `golangci-lint` tool.
+* `build`: Build repository binary in `bin/` directory.
```

Comment:
Question: `./bin/` ?

Reply:
What do you mean? To just add the dot or are you questioning the destination itself?

Reply:
I'm questioning the destination, not if this is right, but if what you mean is:
the binary should be present in `./bin` after a build? Or is this supposed to be `GOPATH/build` ? or some other path?

Reply:
I see. Then my answer is similar to your other point: I am "just" documenting. See [here](https://github.com/mendersoftware/deviceconnect/blob/master/Makefile#L17).

In any case this could use better wording. Do you have a good suggestion for me?

Reply:
Bold leap number 1:
```Build the program binary and output it into the local `bin` directory.```
2.
```Build the project into the `bin` directory```
3.
```Build the repository and output all binaries into `./bin` ```

Reply:
I like `3.` the best. I want to be explicit about binaries as "build the project" might also mean build a Docker image